src/main/java/com/project/javarush/entity/Answer.java

```diff
@@ -21,20 +29,32 @@ public void setAnswerText(String answerText) {
         this.answerText = answerText;
     }
 
-    public long getId() {
+    public Integer getId() {
         return id;
     }
 
-    public void setId(long id) {
+    public void setId(Integer id) {
         this.id = id;
     }
 
-    public long getNextQuestionId() {
+    public Integer getNextQuestionId() {
         return nextQuestionId;
     }
 
-    public void setNextQuestionId(long nextQuestionId) {
+    public void setNextQuestionId(Integer nextQuestionId) {
         this.nextQuestionId = nextQuestionId;
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used
Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement.

the same about other places

Reply:
Done